### DISCUSSION TITLE
How to manage mutiple porject's logs with Seq server

### DISCUSSION THREAD

**Discussion by zhujiancc** [datalust/seq-tickets#1265] [Help]
Can I use workspace to manage different project like this 
![image](https://user-images.githubusercontent.com/19665595/127592351-e35ccb3c-3b3a-4ac5-af0d-237bb55967c5.png)

But how to split logs to different workspace. 

I think that the workspace need a property to filter  logs

**Comment by nblumhardt** [comment]
Hi @zhujiancc - default signals can help with this; if you open "Workspaces" under your avatar:

![image](https://user-images.githubusercontent.com/342712/127597134-cb08b814-4a52-4143-8a09-a1df156eaf51.png)

then under the settings for each workspace you can select default signals:

![image](https://user-images.githubusercontent.com/342712/127597193-ba2b821e-5518-4754-b231-e06f2b30a4ca.png)

These will be activated whenever you enter the workspace.

Let me know if this helps!

Best regards,
Nick

**Reply by zhujiancc** [reply]
Hi @nblumhardt 

Thanks for your answer.

I know "default signals" in workspace, but it  just like a selected tag. 

I hope that I can isolate logs from different department or different projecet, it like pre filter.

Now I can  only add signals with filter Application='ProjectName'  then add other filter.

Is it a good way?

Best regards,
Jian Zhu

**Reply by nblumhardt** [reply]
Thanks for your reply, @zhujiancc.

Using default signals with workspaces is the way to go right now; we have some ideas for a completely isolated "project" type of concept, but that will be some time away, if we do decide to head in that direction.

Hope this helps,
Nick
